.github/workflows/E2E integration.yml

```diff
@@ -81,7 +81,7 @@ jobs:
         continue-on-error: true 
```

Comment:
i feel like this should not continue on error... the check failed on the following step because the sarif was not present

Reply:
proposed in #67